.github/workflows/gitlab-ci.yaml

```
@@ -144,15 +144,15 @@ jobs:
       - name: Run migrations
         run: |
           python manage.py migrate
-
+          
       - name: Run server
         run: |
-          python manage.py runserver 0.0.0.0:8080 &
+          python manage.py runserver 0.0.0.0:8080
 
       - name: Set up Go
         uses: actions/setup-go@v4
```

actionlint (1.7.4)

the runner of "actions/setup-go@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)